run_tests.sh

```diff
@@ -43,18 +43,19 @@ ROOTPATHPATTERN=$(echo $ROOTPATH | sed 's/\\/\\\\/g' | sed 's/\//\\\//g')
 MODPATHS=$(go list -m -f {{.Dir}} all 2>/dev/null | grep "^$ROOTPATHPATTERN")
 for module in $MODPATHS; do
   echo "==> ${module}"
-  env GORACE='halt_on_error=1' go test -v $TESTTAGSWITCH "$TESTTAGS"
-  golangci-lint run --deadline=10m \
-    --disable-all \
-    --enable govet \
-    --enable staticcheck \
-    --enable gosimple \
-    --enable unconvert \
-    --enable ineffassign \
-    --enable structcheck \
-    --enable goimports \
-    --enable misspell \
-    --enable unparam
+  (cd ${module} && \
```

Comment:
A problem with this is that the replace statements in to root module are no longer applied.  The module path being tested would need it's own replaces, but we tend not to do that.  How do we handle this in other repos?

Reply:
@dajohi Any ideas how to address this issue?

Reply:
At least for the tests, https://github.com/decred/dcrd/blob/master/run_tests.sh#L31

I would think the same issue exists for the linter (testing the pinned imports, not the ones in the checked out repo), but I don't know for sure.